app/views/projects/show.html.erb

```diff
@@ -53,12 +52,29 @@ add_project_banner(@project)
                   path: project_members_path(project_id: @project.id,
                                              candidate: @user.id,
                                              target: :project_index)) %>
-<% elsif @project.can_leave?(@user) %>
-  <%= put_button(name: :show_project_leave.t,
-                 class: "btn btn-default btn-sm",
-                 path: project_member_path(project_id: @project.id,
-                                           candidate: @user.id,
-                                           target: :project_index)) %>
+<% else %>
+  <% if @project.trusted_by?(@user) %>
```

Comment:
Question: Would it make sense to move all the conditionals into helpers or otherwise get them out of the template?

Reply:
It would, but I consider that outside the scope of this PR.